# $SHELL is not set, use /bin/sh because it's available
diff --git a/config.toml b/config.toml
index 3768a6e..c323af2 100644
--- a/config.toml
+++ b/config.toml
@@ -6,9 +6,9 @@ vt = 1
 # The default session, also known as the greeter.
 [default_session]
 
-# `agreety` is the bundled agetty/login-lookalike. You can replace `$SHELL`
+# `agreety` is the bundled agetty/login-lookalike. You can replace `/bin/sh`
 # with whatever you want started, such as `sway`.
-command = "agreety --cmd $SHELL"
+command = "agreety --cmd /bin/sh"
 
 # The user to run the command as. The privileges this user must have depends
 # on the greeter. A graphical greeter may for example require the user to be
